datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_profile.py

```diff
@@ -0,0 +1,661 @@
+
+
+    Return a list of SNMP metrics and copy its yaml dump to the clipboard
+    Metric tags need to be added manually
+    """
+    # create a list of all mib files directories and mib names
+    source_directories = set()
+    mibs = set()
+    for file in mib_files:
+        source_directories.add(os.path.dirname(file))
+        mibs.add(os.path.splitext(os.path.basename(file))[0])
+    # create a tmp dir for compiled json mibs
+    json_destination_directory = os.path.join(gettempdir(), 'mibs')
+    if not os.path.exists(json_destination_directory):
+        os.mkdir(json_destination_directory)
+
+    profile_oid_collection = {}
+    # build profile
+    for oid_node in _extract_oids_from_mibs(
+        list(mibs), list(source_directories), json_destination_directory, filter_path_oids
+    ):
+        if oid_node.node_type == 'table':
+            _add_profile_table_node(profile_oid_collection, oid_node)
+        elif oid_node.node_type == 'row':
+            # requires
+            _add_profile_row_node(
+                profile_oid_collection,
+                oid_node,
+                os.path.dirname(mib_files[0]),
+                metric_tag_aliases_path=aliases_metric_tag_path,
+                json_mib_directory=json_destination_directory,
+            )
+        elif oid_node.node_type == 'column':
+            _add_profile_column_node(profile_oid_collection, oid_node)
+        elif oid_node.node_type == 'scalar':
+            _add_profile_scalar_node(profile_oid_collection, oid_node)
+
+    pyperclip.copy(yaml.dump({'metrics': list(profile_oid_collection.values())}, sort_keys=False))
+    print('📋 Profile yaml dump copied to your clipboard')
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Join two SNMP profiles including unique nodes')
+@click.argument('profile_a')
+@click.argument('profile_b')
+@click.pass_context
+def join_profiles(ctx, profile_a, profile_b):
+    """
+    Joins two SNMP profiles, including unique metrics.
+    Returns a list of metrics, and prints its yaml dump.
+    """
+
+    # load data from profile_a
+    profile_a_data = {}
+    _recursively_expand_profile(profile_a, profile_a_data)
+    # index profile data by oids
+    profile_a_oid_collection = _extract_oid_collection_from_profile_data(profile_a_data)
+
+    # load metrics from profile_b
+    profile_b_data = {}
+    _recursively_expand_profile(profile_b, profile_b_data)
+
+    joined_metrics = profile_a_data['metrics']
+    for oid_node in profile_b_data['metrics']:
+        if 'table' in oid_node:
+            oid = oid_node['table']['OID']
+        elif 'symbol' in oid_node:
+            oid = oid_node['symbol']['OID']
+        else:
+            continue
+
+        if oid not in profile_a_oid_collection:
+            joined_metrics.append(oid_node)
+        else:
+            pass
+
+    pyperclip.copy(yaml.dump({'metrics': joined_metrics}, sort_keys=False))
+    print('📋 Joined metrics yaml dump copied to your clipboard')
```

Comment:
I think the output should be sent to stdout instead. To be consistent with unix command line tools. That way we can pipe the output to another command (e.g. cat/grep/etc). 

We can still sent the output to clipboard using pbcopy